tests/components/controlBar-test.js

```diff
@@ -167,7 +167,7 @@ describe('ControlBar', function () {
   it('not render stereo button if content not vr', function () {
     var mockController = {
       state: {
-        isMobile: false,
+        isMobile: true,
```

Comment:
Can we add separate tests for desktop and mobile?

Reply:
This is a test fix, the test was incorrect. Under any circumstances, there is no stereo button on the desktop.